plugins/ps/post-deploy

```diff
@@ -4,5 +4,8 @@ source "$PLUGIN_CORE_AVAILABLE_PATH/common/functions"
 source "$PLUGIN_AVAILABLE_PATH/ps/functions"
 
 APP="$1"
+DOKKU_PROCFILE="$DOKKU_ROOT/$APP/DOKKU_PROCFILE"
+
+[[ -f "$DOKKU_PROCFILE" ]] && rm -f "$DOKKU_PROCFILE"
```

Comment:
It's extracted in pre-deploy currently because `generate_scale_file` which is in pre-deploy now uses the extracted file. Thus it makes sense to do cleanup in post-deploy. We could of course move all of this into deploy in the `dokku` executable, but I don't see a down side of keeping it as it is.